ionic/components/button/button.md.scss

```diff
@@ -58,7 +58,7 @@ $button-md-small-icon-font-size:           1.4em !default;
               color $button-md-transition-duration $button-md-animation-curve;
 
   &:hover:not(.disable-hover) {
-    background-color: $button-md-clear-hover-background-color;
+    background-color: $button-md-color;
```

Comment:
maybe it's just better to remove this entirely? since buttons can be all different colors and perhaps don't need a background color at all on hover or maybe they should lighten the existing button color, regardless of what the color is, with something like `::after { background-color: rgba(white, 0.1); }`